README.md

```diff
@@ -0,0 +1,104 @@
+# setup-nomad-pack
+
+The `hashicorp/setup-nomad-pack` Action sets up the `nomad-pack` CLI in your GitHub Actions workflow by adding the binary to `PATH`.
+
+## Table of Contents
+
+- [setup-nomad-pack](#setup-nomad-pack)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
+
+This GitHub Actions supports all commands that are available in the `nomad-pack` CLI.
+
+The `run`, `destroy`, `info`, and `status` commands require access to a Nomad cluster, as defined through the environment variable `NOMAD_ADDR` (and optionally: `NOMAD_TOKEN`).
```

Comment:
There might need to be other envars set like `NOMAD_CACERT` etc. Probably worth linking to [docs](https://www.nomadproject.io/docs/commands#environment-variables) about this.

Reply:
Great catch! I've reworded this